packages/accounts-controller/src/AccountsController.ts

```diff
@@ -1160,6 +1126,36 @@ export class AccountsController extends BaseController<
     return accountsState;
   }
 
+  /**
+   * Handles the change in multichain network by updating the selected account.
+   *
+   * @param id - The EVM client ID or non-EVM chain ID that changed.
+   */
+  readonly #handleMultichainNetworkChange = (
```

Comment:
The handler name is not following the same convention as the others, it should be `#handleOnMultichainNetworkDidChange`

Reply:
Updated